src/main/java/com/owncloud/android/lib/common/OwnCloudClient.java

```diff
@@ -63,7 +63,7 @@
     /**
      * Characters to skip during userID encoding
      */
-    private static final String ALLOWED_USERID_CHARACTERS = "@";
+    private static final String ALLOWED_USERID_CHARACTERS = "@+";
```

Comment:
Can you add 1-2 tests for this?

Reply:
Done https://github.com/nextcloud/android-library/pull/792/commits/2fa7206cf7dbb92c3b4438b9638a892b1ba85814